### DISCUSSION TITLE
Line height in config changes gap between tab bar. Best way to minimize?

### DISCUSSION THREAD

**Discussion by benfrain** [kovidgoyal/kitty#5077] [Q&A]
Currently there is a gap top and bottom of the actual Kitty area (set that to #aaa merely to illustrate). The image shows it below neovim.

<img width="419" alt="image" src="https://user-images.githubusercontent.com/1318466/167427567-e96d3335-43bf-412a-9637-fe958d190f19.png">

I've noticed this gets larger when I change my line height. Having[ read the docs](https://sw.kovidgoyal.net/kitty/conf/#window-layout) I suspect this is something to do with how many cells the area is divisible by?

I have tried setting `resize_in_steps yes` but this doesn't seem to make any difference. Are there any ways to get this so there is no gap? 

I also wondered if my screen height / a cleanly divisible number may help but that didn't seem to help (eg 1440 vertical height / 18px font size)

**Comment by kovidgoyal** [comment]
You will have no gaps if your kitty window size is an exact multiple of the
cell height.
